tdrs-backend/docker-compose.local.yml

```diff
@@ -82,7 +82,7 @@ services:
                ./gunicorn_start.sh && 
                celery -A tdpservice.settings worker -l info"
     ports:
-      - "8080:8080"
+      #- "8080:8080"
```

Comment:
Should this just be removed instead of commented out for both docker-compose files?